lib/internal/cluster/master.js

```diff
@@ -70,6 +79,15 @@ cluster.setupMaster = function(options) {
   assert(schedulingPolicy === SCHED_NONE || schedulingPolicy === SCHED_RR,
          `Bad cluster.schedulingPolicy: ${schedulingPolicy}`);
 
+  const hasDebugArg = process.execArgv.some((argv) => {
```

Comment:
@refack It's more funny if you know that `enbale` phonetically is similar to swearing in Russian, which used by someone in situations when he got upset.